widget/playground/src/containers/StyleLayout/StyleLayout.General.tsx

```diff
@@ -113,6 +131,19 @@ export function General() {
           />
         </OverlayPanel>
       )}
+
+      {modalState === ModalState.DEFAULT_LANGUAGE && (
+        <OverlayPanel onBack={onBack}>
+          <SingleList
+            onChange={handleLanguageChange}
+            title="Default Language"
```

Comment:
It should be "Languages"

Reply:
fixed

Reply:
I meant just "Languages" not "Default Languages". based on Figma.